### PR TITLE
UI Improvements following project's website

### PR DESCRIPTION
**Pull request by louisgrasset**
What's new ? 💄 
-----
+ Switch to global green theme from [cuberite](https://cuberite.org)
+ [Nanobar](https://github.com/jacoborus/nanobar) Implementation on each webadmin page

Few adjustments : 🖊 
-----
+ Google fonts fixed
+ CSS simplification (see common.css)
+ CSS / HTML indentation fixed

image

### CODE REVIEW

File: Server/webadmin/template.lua [hunk 1]
```diff
@@ -1,103 +1,103 @@
 -- Use a table for fast concatenation of strings
 local SiteContent = {}
 function Output(String)
-	table.insert(SiteContent, String)
+    table.insert(SiteContent, String)
```
Comment (lkolbly) [inline Server/webadmin/template.lua]: Indentation should be tabs.

File: Server/webadmin/files/style.css [hunk 2]
```diff
@@ -324,39 +135,28 @@ table {
 }
 
 table th {
-    border-bottom: 1px solid rgba(0,0,0,0.12);
+    border-bottom: 1px solid rgba(0, 0, 0, 0.12);
```
Comment (lkolbly) [inline Server/webadmin/files/style.css]: Spaces after commas, I like.

**Comment by sphinxc0re** [comment]
@louisgrasset Why did you close this. It seemed promising 😉

**Comment by mathiascode** [comment]
Another PR for a webadmin redesign was opened earlier #3968

**Pull request closed without merging**